_includes/team_member.html

```diff
@@ -0,0 +1,24 @@
+<div class="row justify-content-center align-items-center">
+    <div class="col-md-3 col-md-offset-2">
+        <div class="team-pic text-center" id="{{ team.twitter }}">
+            <figure><img src="{{ "/assets/team/" | prepend: site.url }}{{ team.pic | default: 'placeholder.png' }}" alt="Image" class="rounded-circle img-fluid"></figure>
```

Comment:
@kennyhong We should do srcset to serve up a high res pic and a lo res pic. Not a big deal now. This can be done later